src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Better Bayesian Workfllows with Inference Data and Xarray'
+date: '2022-12-17'
+authors:
+  - name: Ravin Kumar
+    github: canyon289
+  - name: Justus Magin
+    github: keewis
+summary: 'Xarray now supports unit-aware operations by wrapping pint arrays'
+---
+
+_TLDR: Xarray is the core of the `az.InferenceData` object. [ArviZ InferenceData](INSERT LINK) simplifies the Bayesian workflow, facilitates reproducibility, and enables interoperability between different Probabilistic Programming languages._
```

Comment:
```suggestion
_TLDR: Xarray is the core of the `az.InferenceData` object. [ArviZ InferenceData](https://python.arviz.org/en/stable/api/generated/arviz.InferenceData.html) simplifies the Bayesian workflow, facilitates reproducibility, and enables interoperability between different Probabilistic Programming languages._
```

Is this the link you want? Or perhaps https://python.arviz.org/en/stable/schema/schema.html#terminology would be better?